CONTRIBUTING.rst

```diff
@@ -81,20 +85,22 @@ and running locally::
 
 Then run the initial migrations::
 
-    python manage.py migrate
+    ./manage.py migrate
 
 Run this command if you would like to populate the database with fake testing
 data:
 
-    python manage.py populate_db
+    ./manage.py populate_db
 
 Run this command to create a necessary repeatable django-rq job in the database::
 
     ./manage.py metaci_scheduled_jobs
 
 
-Copying the production database
--------------------------------
+Alternative: Copying the production database
+---------------------------------------------
+
+** PLEASE CONTACT THE METACI TEAM BEFORE DOING THIS TO DISCUSS THE SECURITY IMPLICATAIONS **
```

Comment:
Maybe a `WARNING: (...)` will suffice? Noting that production secrets are included in the copy and warning users to beware is more generally applicable to external users.

Reply:
@davisagli We are leaning towards just taking this section out.

Reply:
Yeah let's take it out.

Reply:
Done.

https://github.com/SFDO-Tooling/MetaCI/pull/414/commits/036996e0d0b1ab266421eeec73a8f575c200a6e7